smart_ats/jobs/api/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import permissions
+
+
+class IsJobAuthorOrReadOnly(permissions.BasePermission):
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+        if not hasattr(request.user, "companyadmin"):
+            return False
+        if request.method == "POST" and not obj:
```

Comment:
with this scenario, if the user sends a company that is not related to him, it will create the job anyway. This is wrong.